datafusion/sql/Cargo.toml

```diff
@@ -46,6 +46,7 @@ arrow-array = { workspace = true }
 arrow-schema = { workspace = true }
 datafusion-common = { workspace = true, default-features = true }
 datafusion-expr = { workspace = true }
+datafusion-functions-aggregate= { workspace = true }
```

Comment:
This is probably not desired. Needed because of this function:

https://github.com/eejbyfeldt/datafusion/blob/b42c70bac4af5ef17ce24ca1a2e95efa3f7cece9/datafusion/sql/src/expr/mod.rs#L596-L626

The comment claims to that it should be possible to move to ArrayAgg::simplify, will need to look more into that API.

Reply:
I am not sure how it would be possible to do with the current simplify API.